tests/unit/helpers.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Helper functions used to unit test Jenkins charm."""
+
+from pathlib import Path
+
+import requests
+
+
+def make_relative_to_path(tmp_path: Path, root_path: Path) -> Path:
```

Comment:
What is the reason for needing this please?

Reply:
Removed w/ refactored Harness container!